source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj

```diff
@@ -2327,7 +2327,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				HEADER_SEARCH_PATHS = "";
-				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
Is there any reason to update this?

Reply:
yes, Team was the bottleneck on iOS version upgrade, and they bumped to ios 14, so we are bumping it, too.

Reply:
The team was the bottleneck on the ios version. They bumped their version to 14.0, so I bumped it up.